src/canvas/Curve.tsx

```diff
@@ -20,6 +20,19 @@ export class Curve {
         return resultingPoints;
     };
 
+    getLength = (): number => {
+        let length = 0;
+        // compute the total distance from start to control + from control to end in order to compute an 
+        // appropriate number of points on the curve.
+        const NUMPOINTS = Math.ceil((Math.sqrt(this.start.distanceSquared(this.control)) + Math.sqrt(this.control.distanceSquared(this.end))) / 5);
+        const pointsOnCurve = this.computePointsOnCurve(NUMPOINTS);
+        for (let i = 0; i < NUMPOINTS - 1; i++) {
+            length += Math.sqrt(pointsOnCurve[i+1].distanceSquared(pointsOnCurve[i]));
+        }
+        return length;
+    }
+
+    // using De Casteljau's algorithm (slower but more stable than the direct approach)
```

Comment:
What is this comment referring to?

Reply:
The algorithm used in computePoints is a known algorithm. I thought it would be useful to document the name of the algorithm as well as why it was used instead of a direct computation.